releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -443,7 +465,7 @@
   "122892": {
     "commit": "573832a42f03af912a5611eb4e81a0e812a5ddf2",
     "text": "The \"fake\" clients generated by `client-gen` now have the same semantics on\nerror as the real clients; in particular, a failed Get(), Create(), etc, no longer\nreturns `nil`. (It now returns a pointer to a zero-valued object, like the real\nclients do.) This will break some downstream unit tests that were testing\n`result == nil` rather than `err != nil`, and in some cases may expose bugs\nin the underlying code that were hidden by the incorrect unit tests.",
-    "markdown": "The \"fake\" clients generated by `client-gen` now have the same semantics on\n  error as the real clients; in particular, a failed Get(), Create(), etc, no longer\n  returns `nil`. (It now returns a pointer to a zero-valued object, like the real\n  clients do.) This will break some downstream unit tests that were testing\n  `result == nil` rather than `err != nil`, and in some cases may expose bugs\n  in the underlying code that were hidden by the incorrect unit tests. ([#122892](https://github.com/kubernetes/kubernetes/pull/122892), [@danwinship](https://github.com/danwinship)) [SIG API Machinery, Auth, Cloud Provider, Instrumentation and Storage]",
+    "markdown": "The \"fake\" clients generated by `client-gen` now have the same semantics on\n  error as the real clients; in particular, a failed Get(), Create(), etc, no longer\n  returns `nil`. (It now returns a pointer to a zero-valued object, like the real\n  clients do.) This will break some downstream unit tests that were testing\n  `result == nil` rather than `err != nil`, and in some cases may expose bugs\n  in the underlying code that were hidden by the incorrect unit tests. ([#122892](https://github.com/kubernetes/kubernetes/pull/122892), [@danwinship](https://github.com/danwinship))",
```

Comment:
```suggestion
    "markdown": "The \"fake\" clients generated by `client-gen` now have the same semantics on\n  error as the real clients; in particular, a failed Get(), Create(), etc, no longer\n  returns `nil`. (It now returns a pointer to a zero-valued object, like the real\n  clients do.) This will break some downstream unit tests that were testing\n  `result == nil` rather than `err != nil`, and in some cases may expose bugs\n  in the underlying code that were hidden by the incorrect unit tests. ([#122892](https://github.com/kubernetes/kubernetes/pull/122892), [@danwinship](https://github.com/danwinship)) [SIG API Machinery, Auth, Cloud Provider, Instrumentation and Storage]",
```